src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -387,7 +441,8 @@ def main():
         outdir = outdir + "/"
 
     if not os.path.exists(outdir):
-        raise FileNotFoundError("No such outputdirectory:", outdir)
+        os.mkdir(outdir)
```

Comment:
Implicit mkdir is against current recommended practice..

Reply:
I gave in to customer request. Agree, will revert